worlds/LauncherComponents.py

```diff
@@ -116,6 +116,8 @@ def _install_apworld(apworld_src: str = "") -> Optional[Tuple[pathlib.Path, path
     import shutil
     shutil.copyfile(apworld_path, target)
 
+    worlds.WorldSource(str(target), is_zip=True).load()
```

Comment:
If you drop the same file twice, this raises an exception for "already registered". I'm wondering how we should handle that?

Reply:
This went through a lot to address.
The `WorldSources` is now added to `world_sources` so we can check if it's already installed.
If a world with the same name is already loaded, it will show a message to the user that the Launcher needs to be restarted to use the new world installation.